ci/serverless/test_data/index_template.json

```diff
@@ -4,8 +4,7 @@
   "priority": 500,
   "template": {
     "settings": {
-      "index.default_pipeline": "integration-logstash_test.events-default",
-      "index.lifecycle.name": "logs"
+      "index.default_pipeline": "integration-logstash_test.events-default"
```

Comment:
Any reason why the `index.lifecycle.name` was removed?

Reply:
This config does not exist in serverless. It breaks everything in QA.